sources/core/Stride.Core.Mathematics/Vector3.cs

```diff
@@ -1507,6 +1552,19 @@ public static void RotationYawPitchRoll(ref Quaternion quaternion, out Vector3 y
             return new Vector3(left.X * right.X, left.Y * right.Y, left.Z * right.Z);
         }
 
+        /// <summary>
+        /// Return the vector rotated by the quaternion.
+        /// </summary>
+        /// <remarks>
+        /// Shorthand for <see cref="Quaternion.Rotate(ref Vector3)"/>
+        /// </remarks>
+        public static Vector3 operator *(in Vector3 left, in Quaternion right)
```

Comment:
It would be confusing to have two separate PRs/commits for the same thing, I'll cover this with the other one I have in my PR.